test/serialize.js

```diff
@@ -5,6 +5,15 @@ var cookie = require('..');
 
 suite('serialize');
 
+test('argument validation', function() {
+    var nonStrings = [1010, true, {}, (function() {return;}), new Date(), []];
```

Comment:
Please just do one `assert.throws` line for each instead of iterating an array; an error won't tell you which value was allowed though, only point to line 12 or 13, which is not helpful.